Libraries/Text/Text.js

```diff
@@ -317,6 +326,12 @@ const Text = React.createClass({
         isHighlighted: this.state.isHighlighted,
       };
     }
+    if (typeof newProps.selectionColor !== 'undefined') {
```

Comment:
Looking at other examples (TextInput) it seems like we don't need to manually call processColor for props of type Color.